### PR TITLE
Move buzz dependance to suggest section

### PR DESCRIPTION
**Pull request by marmotz**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,8 +11,10 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "2.*",
-        "kriswallsmith/buzz": "dev-master"
+        "symfony/framework-bundle": "2.*"
+    },
+    "suggest": {
+        "kriswallsmith/buzz": "Allows you to use Buzz adapter"
```
Comment (geoffrey-brier) [inline composer.json]: I think you should add `"ext-curl"`

Comment (marmotz) [inline composer.json]: yep, done :)

**Comment by geoffrey-brier** [comment]
That's good for me, thanks :)

**Comment by marmotz** [comment]
Thanks !

**Comment by egeloen** [comment]
:+1:

**Pull request merged**